packages/in_app_purchase/in_app_purchase_android/android/src/main/java/io/flutter/plugins/inapppurchase/Translator.java

```diff
@@ -233,6 +235,34 @@ static HashMap<String, Object> fromBillingResult(BillingResult billingResult) {
     return info;
   }
 
+  static HashMap<String, Object> fromUserChoiceDetails(UserChoiceDetails userChoiceDetails) {
```

Comment:
Nit: add docs links to the places we are essentially copying the structure of android classes here, so 
https://developer.android.com/reference/com/android/billingclient/api/UserChoiceDetails#getOriginalExternalTransactionId() here 
and 
https://developer.android.com/reference/com/android/billingclient/api/UserChoiceDetails.Product
below